README.md

```diff
@@ -635,6 +635,51 @@ If you use `zf-apigility-admin`'s Admin API and/or the Apigility UI to
 configure authentication adapters, the above configuration will be created for
 you.
 
+#### Attach a custom storage adapter
```

Comment:
See https://github.com/API-Skeletons/zf-oauth2-doctrine/blob/master/README.md#configuration-with-zfcampuszf-mvc-auth and the section below it on zf-oauth integration